root/report/ReleasesWithAmazonCoverArt.js

```diff
@@ -24,9 +24,9 @@ const ReleasesWithAmazonCoverArt = ({
     canBeFiltered={canBeFiltered}
     description={l(
       `This report shows releases which have cover art from Amazon, but have
-       no front cover in the Cover Art Archive. The use of Amazon art
-       is going to be discontinued eventually, so these releases will lose
-       their front cover unless one is added to the Cover Art Archive.`,
+       no front cover in the Cover Art Archive. The use of Amazon art has been
+       discontinued since the 16th of May 2022, so these releases have no
+       front cover unless one is added to the Cover Art Archive.`,
```

Comment:
What about something like:

```
      `This report shows releases which have cover art on Amazon, but have
       no front cover in the Cover Art Archive. The use of Amazon art has been
       discontinued since the 16th of May 2022, so these releases have no
       front cover art anymore until one is added to the Cover Art Archive.`,
```

Reply:
("from Amazon" suggests they still have it, and I feel "until" is more clear than "unless" about what we expect to happen :) )

Reply:
Included your changes in commit https://github.com/metabrainz/musicbrainz-server/pull/2379/commits/1600f4938060f7caf7bd8c19ae0b82fad7c94d78